app/web/features/connections/friends/useFriendRequests.test.tsx

```diff
@@ -172,13 +164,10 @@ describe("when the listFriendRequests succeeds", () => {
     });
   });
 
-  it("returns isError as true with the error and missing friend data in request object of response if some getUser queries failed", async () => {
-    getUserMock.mockImplementation((userId: string) => {
-      return userId === "3"
-        ? Promise.reject(new Error(`Error fetching user ${userId}`))
-        : getUser(userId);
-    });
-    jest.spyOn(console, "error").mockReturnValue(undefined);
+  it("returns isError as true with the error if some getLiteUsers query failed", async () => {
```

Comment:
```suggestion
  it("returns isError as true with the error if the getLiteUsers query failed", async () => {
```
With the refactored endpoint, isn't this always going to be a single query? The phrasing in the old test ("some getUser queries") is alluding to the fact it was making multiple requests which I believe is no longer possible?

Reply:
Ah yeah copy pasta good catch!